<?xml version="1.0"?>
<launch>

	<node name="heightmap_node" pkg="velodyne_height_map" type="heightmap_node" >
		<remap from="velodyne_points" to="perfect_velodyne/normal" />
	</node>

	<node pkg="local_tutorials" type="ovg" name="OccupancyVoxelGrid" output="screen">
		<param name="max_x_range" value="50.0"/>
		<param name="max_y_range" value="50.0"/>
		<param name="max_z_range" value="10.0"/>
		<param name="voxel_size" value="0.2"/>
		<param name="SKIP" value="5"/>
		<param name="STEP" value="10"/>
		<param name="probability_threshold" value="0.7"/>
		<param name="save_raw_flag" value="0"/>
		<param name="debug_pub_flag" value="0"/>
	
		<remap from="velodyne_points" to="perfect_velodyne/normal" />
		<!-- <remap from="velodyne_points" to="velodyne_obstacles" /> -->
	</node>
	
</launch>
